pkg/cmd/roachtest/main.go

```diff
@@ -336,6 +336,11 @@ func testShouldBeSkipped(
 		// test should not be skipped for this suite
 		return false
 	}
+
+	if test.Randomized {
```

Comment:
Nit: maybe move up since it short circuits the iteration logic above?

Reply:
Done.